app-misc/anki/anki-2.1.15.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_REQ_USE="sqlite"
+
+inherit desktop python-single-r1 xdg
+
+DESCRIPTION="A spaced-repetition memory training program (flash cards)"
+HOMEPAGE="https://apps.ankiweb.net"
+SRC_URI="https://apps.ankiweb.net/downloads/current/${P}-source.tgz -> ${P}.tgz"
+
+S="${WORKDIR}/${P}"
+
+LICENSE="AGPL-3+ BSD MIT GPL-3+ CC-BY-SA-3.0 Apache-2.0 CC-BY-2.5"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="latex +recording +sound test"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		>=dev-python/PyQt5-5.12[gui,svg,widgets,${PYTHON_MULTI_USEDEP}]
+		>=dev-python/PyQtWebEngine-5.12[${PYTHON_MULTI_USEDEP}]
+		>=dev-python/httplib2-0.7.4[${PYTHON_MULTI_USEDEP}]
+		dev-python/beautifulsoup:4[${PYTHON_MULTI_USEDEP}]
+		dev-python/decorator[${PYTHON_MULTI_USEDEP}]
+		dev-python/jsonschema[${PYTHON_MULTI_USEDEP}]
+		dev-python/markdown[${PYTHON_MULTI_USEDEP}]
+		dev-python/requests[${PYTHON_MULTI_USEDEP}]
+		dev-python/send2trash[${PYTHON_MULTI_USEDEP}]
+		recording? (
+			dev-python/pyaudio[${PYTHON_MULTI_USEDEP}]
+			media-sound/lame
+		)
+	')
+	sound? ( || (
```

Comment:
Conditional runtime dependencies are [forbidden](https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001) in general. Usually, optfeature from the eutils [eclass](https://devmanual.gentoo.org/eclass-reference/eutils.eclass/) should be used instead in pkg_postinst.

Reply:
The use of the  `sound` flag in what looks like the same way goes back to at least the start of the git history.  Given the long pre-existing nature I'd prefer to defer figuring out how to handle this to a separate ticket.

Tangent: How did you know it was a conditional runtime dependency just from looking at the ebuild?